src/utils/viewer/viewer.cpp

```diff
@@ -213,139 +199,115 @@ Viewer::Viewer(const Arguments& arguments)
   Mn::GL::Renderer::enable(Mn::GL::Renderer::Feature::DepthTest);
   Mn::GL::Renderer::enable(Mn::GL::Renderer::Feature::FaceCulling);
 
-  int sceneID = sceneManager_.initSceneGraph();
-  sceneID_.push_back(sceneID);
-  sceneGraph_ = &sceneManager_.getSceneGraph(sceneID);
-  rootNode_ = &sceneGraph_->getRootNode();
-
-  stageFileName = args.value("scene");
-  esp::assets::AssetInfo info = esp::assets::AssetInfo::fromPath(stageFileName);
-  std::string sceneLightSetup = esp::assets::ResourceManager::NO_LIGHT_KEY;
-  if (args.isSet("scene-requires-lighting")) {
-    info.requiresLighting = true;
-    sceneLightSetup = esp::assets::ResourceManager::DEFAULT_LIGHTING_KEY;
+  sceneFileName = args.value("scene");
+  bool useBullet = args.isSet("enable-physics");
+  if (useBullet && (args.isSet("debug-bullet"))) {
+    debugBullet_ = true;
   }
 
-  std::string physicsConfigFilename = args.value("physics-config");
-  if (!Cr::Utility::Directory::exists(physicsConfigFilename)) {
-    LOG(FATAL)
-        << physicsConfigFilename
-        << " was not found, specify an existing file in --physics-config";
+  // configure and intialize Simulator
+  auto simConfig = esp::sim::SimulatorConfiguration();
+  simConfig.scene.id = sceneFileName;
+  simConfig.enablePhysics = useBullet;
+  simConfig.frustumCulling = true;
+  if (args.isSet("stage-requires-lighting")) {
+    Cr::Utility::Debug() << "Stage using DEFAULT_LIGHTING_KEY";
```

Comment:
```suggestion
    Mn::Debug{} << "Stage using DEFAULT_LIGHTING_KEY";
```

is shorter and alias to the same thing, if you prefer. Same elsewhere